ext/mbstring/mbstring.stub.php

```diff
@@ -65,29 +65,29 @@ function mb_list_encodings(): array {}
 
 function mb_encoding_aliases(string $encoding): array {}
 
-function mb_encode_mimeheader(string $string, ?string $charset = null, ?string $transfer = null, string $linefeed = "\r\n", int $indent = 0): string {}
+function mb_encode_mimeheader(string $string, ?string $charset = null, ?string $transfer_encoding = null, string $linefeed = "\r\n", int $indent = 0): string {}
 
 function mb_decode_mimeheader(string $string): string {}
 
-function mb_convert_kana(string $string, string $option = "KV", ?string $encoding = null): string {}
+function mb_convert_kana(string $string, string $mode = "KV", ?string $encoding = null): string {}
 
-function mb_convert_variables(string $to, array|string $from, mixed &$var, mixed &...$vars): string|false {}
+function mb_convert_variables(string $to_encoding, array|string $from_encoding, mixed &$var, mixed &...$vars): string|false {}
```

Comment:
Suggestion:

`$from` / `$to` -> `$decode` / `$encode`

Reply:
Not sure I understand the suggestion. Are you saying the parameter names should be `$encode` and `$decode` or `$encode_encoding` and `$decode_encoding`? Both don't really make sense to me.

Reply:
My suggestion was to name them `$encode` and `$decode`. My idea was that `$decode` says how to decode the string while `$encode` says how to encode it again. It made me think of how it would be written in Python:

```my_string.decode('iso-8859-1').encode('utf8')```

The names are short too. To me it does make sense, but that could just be me. :)

Reply:
I'm happy to leave these at just `$to` and `$from` if the `_encoding` suffix makes them too long.

Reply:
P.S. I like their current names ^^ But unfortunately I can only have a closer look a bit later.

Reply:
Just to mention a few other options: `UConverter::__construct()` uses `$source_encoding` and `$destination_encoding` (`$target_encoding` reads better for me though).